### ISSUE TITLE
Consideration; show what information each tracker is collecting.

### ISSUE THREAD

**Issue by reZach** [Dan-inpooling/Privacy-eye#1]
Can Privacy-eye list out what pieces of personal information these trackers send? As a visual learner, it would be helpful (and good to callout) "First name", "Address", "Zip", "IP" as visual callouts on the screen. Doing so would help users more-easily understand what's happening when these trackers load.

**Comment by Dan-inpooling** [comment]
Good point! I debated on this before. We want it to run locally/offline. Technically we can't see what info collected by each tracker. But most trackers (es. ad trackers) are collecting similar types info. We can maybe add a banner to show how tracker works and what general type of info they collect - being informational.

**Comment by reZach** [comment]
Is it possible to go to these companies' websites and into the privacy policies and collect that information once, with a timestamp. That data can remain "offline" and simply be packaged with the extension? It's definitely _possilble_.

**Comment by Dan-inpooling** [comment]
It's possible. I even went thru a couple of API documents to see the data collected. But most of time, they are just collecting everything you can imagine ...and quite standard... It can be quite time consuming. But if anyone who wants to help on this. We will be very glad to do so.
